packages/gatsby-link/src/__tests__/index.js

```diff
@@ -19,6 +19,12 @@ const getNavigateTo = () => {
 }
 
 describe(`<Link />`, () => {
+
+  it(`does not fail to initialize when __PREFIX_PATHS__ is not defined`, () => {
```

Comment:
@DSchau updated the test.  It does give nicer feedback when the test fails in that format.  Good suggestion.